src/components/QRcode.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+<div>
```

Comment:
if there's only 1 canvas then maybe no need to wrap inside a <div>

Reply:
I tried. but can't remove this